handler/oauth2/flow_authorize_jwt_session.go

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
What do you think about moving these from `/oauth2/` to `/rfc7523/`?

Reply:
Don't quiet understand why you want to move this particular file to rfc7523, rfc7523 is also about oauth2.0. Can you please elaborate your intention?